src/flb_utils.c

```diff
@@ -1014,16 +1014,30 @@ int flb_utils_write_str_buf(const char *str, size_t str_len, char **out, size_t
     return 0;
 }
 
-static char *flb_copy_host(const char *string, int pos_init, int pos_end)
+char *flb_copy_host(const char *string, int pos_init, int pos_end)
```

Comment:
if this interface will be from public use (nonstatic), the name must be prefixed according to the file, e.g: flb_utils_host_copy()